packages/enzyme-adapter-utils/src/Utils.js

```diff
@@ -94,6 +97,10 @@ export function displayNameOfNode(node) {
 
   if (!type) return null;
 
+  if (type === Portal) {
+    return 'Portal';
```

Comment:
i can't decide if it's better to add this here, or to require that adapters implement `displayNameOfNode` in order to support portals?